torch/profiler/profiler.py

```diff
@@ -16,16 +16,18 @@ class ProfilerAction(Enum):
     RECORD_AND_SAVE = 3
 
 
-def schedule(*, wait: int, warmup: int, active: int) -> Callable:
+def schedule(*, wait: int, warmup: int, active: int, repeat: int = 0) -> Callable:
     """
     Returns a callable that can be used as profiler ``schedule`` argument. The profiler will wait for ``wait`` steps, then
     do the warmup for the next ``warmup`` steps, then
     do the active recording for the next ``active`` steps and then
-    repeat the cycle staring with the next step.
+    repeat the cycle starting with the next step and end after ``repeat`` cycles.
```

Comment:
repeat the cycle starting with the next step. The number of cycles is specified by the ``repeat`` parameter. When the parameter's value is zero, the cycles will continue until the profiling is finished.